deploy/examples/import-external-cluster.sh

```diff
@@ -34,6 +35,12 @@ function checkEnvVars() {
   if [ -z "$NAMESPACE" ]; then
     echo "Please populate the environment variable NAMESPACE"
     exit 1
+  fi
+  if [ -z "$NAMESPACE" ]; then
```

Comment:
copy-paste error? :) 
```suggestion
  if [ -z "$ROOK_RBD_FEATURES" ]; then
```

Reply:
🤐